hack/generate-install.sh

```diff
@@ -74,10 +74,30 @@ kubectl kustomize config/clusterresources | sed s/:latest/:$TAG/ > $RUNTIMES_INS
 # Update ingressGateway in inferenceservice configmap as 'kubeflow/kubeflow-gateway'
 yq -i 'select(.metadata.name == "inferenceservice-config").data.ingress |= (fromjson | .ingressGateway = "kubeflow/kubeflow-gateway" | tojson)' $KUBEFLOW_INSTALL_PATH
 
+# Create a temp directory for final CRD
+mkdir config/crd_helm
+delimeter_lines=$(cat ${INSTALL_PATH} -n |grep '\-\-\-'|cut -f1)
```

Comment:
this works on Linux but does not work on Mac OS can you swap the `-n` and the `${INSTALL_PATH}`?


```sh
$ INSTALL_PATH=./install/v0.12.0-rc1/kserve.yaml

$ cat ${INSTALL_PATH} -n |grep '\-\-\-'|cut -f1

cat: -n: No such file or directory
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---
---


[kserve] ((v0.12.0-rc1) *)$ cat -n ${INSTALL_PATH} | grep '\-\-\-' | cut -f1

     9
  1904
  2568
  3099
 18815
 20710
 20821
 20831
 20873
 21178
 21196
 21210
 21223
 21236
 21585
 21591
 21612
 21629
 21641
 21731
 21745
 21753
 21812
 21842
 21872
 21902
 21932
```

Reply:
updated